prime-e2e-tests/Enrollment.Tests.cs

```diff
@@ -12,66 +14,155 @@ public class EnrollmentTests : BaseTest
         [Test]
         public void FullFamilyPhysician_HappyPath()
         {
+            Person enrollee = new Person();
+
             _driver.Navigate().GoToUrl(TestParameters.EnrollmentUrl);
 
             LoginWithBCSC();
 
-            // Perform enrollment
+            CompleteCollectionNoticePage();
+
+            CompleteEnrolleeInformationPage(enrollee);
+
+            CompleteCareSettingPage_CommunityPharmacy();
+
+            CompleteCollegeLicencePage_FullFamilyPhysician();
+
+            CompleteSelfDeclarationPage();
+
+            CompleteEnrolmentReviewPage();
+        }
+
+
+        [Test]
+        public void MedicalOfficeAssistant_HappyPath()
+        {
+            Person enrollee = new Person();
+
+            _driver.Navigate().GoToUrl(TestParameters.EnrollmentUrl);
 
-            Assert.AreEqual("Collection of Personal Information Notice", _driver.FindPatiently("//h1[@class='mb-4']").Text);
-            _driver.TakeScreenshot("Collection_Notice");
+            LoginWithBCSC();
+
+            CompleteCollectionNoticePage();
+
+            CompleteEnrolleeInformationPage(enrollee);
+
+            CompleteCareSettingPage_CommunityPharmacy();
```

Comment:
My test failed at this line w/ a fresh enrolment. Is there any prerequisite needs to be meet before running the test?

`
Starting test execution, please wait...
A total of 1 test files matched the specified pattern.
  Failed MedicalOfficeAssistant_HappyPath [20 s]
  Error Message:
     Expected: True
  But was:  False

  Stack Trace:
     at TestPrimeE2E.Enrollment.EnrollmentTests.CompleteCareSettingPage_CommunityPharmacy() in C:\Users\177054\source\moh-prime\prime-e2e-tests\Enrollment.Tests.cs:line 86
   at TestPrimeE2E.Enrollment.EnrollmentTests.MedicalOfficeAssistant_HappyPath() in C:\Users\177054\source\moh-prime\prime-e2e-tests\Enrollment.Tests.cs:line 50
`

Reply:
That's strange, it works for me just now against localhost and this branch `feature/PRIME-1549-remove-job-references` (up-to-date with origin).  We could meet on Teams about it @johnnyl6  if still a problem.